### PR TITLE
Update README with sample configuration

### PR DESCRIPTION
**Pull request by naveentatikonda**
Signed-off-by: Naveen Tatikonda <navtat@amazon.com>

### Description
Add sample output configuration to README

### Issues Resolved
https://github.com/opensearch-project/logstash-output-opensearch/issues/79

### Check List
- [x] Commits are signed as per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 [![Build and Test logstash-output-opensearch plugin](https://github.com/opensearch-project/logstash-output-opensearch/actions/workflows/CI.yml/badge.svg)](https://github.com/opensearch-project/logstash-output-opensearch/actions/workflows/CI.yml)
 ![PRs welcome!](https://img.shields.io/badge/PRs-welcome!-success)
-# Logstash Plugin
+# Logstash Output Opensearch
```
Comment (VijayanB) [inline README.md:3]:
```suggestion
# Logstash Output OpenSearch
```

**Review by VijayanB — APPROVED** [review]

**Review by vamshin — COMMENTED** [review]
Shall we add one example for using DataStreams?

**Comment by naveentatikonda** [comment]
> Shall we add one example for using DataStreams?

Added sample configuration for data streams.

**Review by martin-gaievski — APPROVED** [review]

**Comment by dlvenable** [comment]
Thanks @naveentatikonda !

**Pull request merged**